src/utils/shared.py

```diff
@@ -89,4 +89,4 @@ def set_order(objects, order_attr_name, pk_list):
 
   
 def day_month(date):
-    return date.strftime("%d-%b")
+    return date.strftime("%d-%m")
```

Comment:
The original formatting was set in ff3d7e8bb since it was confusing for thoe in countries where the default date  is represented as month-date.

I'd advise reverting this change

Reply:
I think if we revert this we should have the due date match so there aren't two weird date formats right next to one another?